concurrency-limits-core/src/test/java/com/netflix/concurrency/limits/limit/VegasLimitTest.java

```diff
@@ -58,7 +58,7 @@ public void noChangeIfWithinThresholds() {
     @Test
     public void decreaseSmoothing() {
         VegasLimit limit = VegasLimit.newBuilder()
-            .decrease(current -> current / 2)
+            .decreaseFunction(current -> current / 2)
```

Comment:
nit: can we test both `.decrease` and `.decreaseFunction` to ensure we haven't introduced a regression with the existing code? same in the `decreaseWithoutSmoothing()` test.

Reply:
Okay, I've added tests for `decrease`. A lot of the other builder methods don't seem to have coverage, could probably try tackling that in a follow up.